### DISCUSSION TITLE
PaddleOCR/latest/applications/%E5%8D%B0%E7%AB%A0%E5%BC%AF%E6%9B%B2%E6%96%87%E5%AD%97%E8%AF%86%E5%88%AB

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14140] [Q&A]
# PaddleOCR/latest/applications/%E5%8D%B0%E7%AB%A0%E5%BC%AF%E6%9B%B2%E6%96%87%E5%AD%97%E8%AF%86%E5%88%AB

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/applications/%E5%8D%B0%E7%AB%A0%E5%BC%AF%E6%9B%B2%E6%96%87%E5%AD%97%E8%AF%86%E5%88%AB.html

**Comment by Ethenlalala** [comment]
AI studio项目未公开

**Comment by cuicheng01** [comment]
您好，这个是之前的范例，不是PaddleOCR官方的模型。当然，印章识别可以考虑参考PaddleX的[印章文本识别产线](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/pipeline_usage/tutorials/ocr_pipelines/seal_recognition.md)，官方支持，精度更高。
